deployer/README.md

```diff
@@ -8,7 +8,7 @@ To create the necessary AWS resources you can use GeoEngineer which requires `ru
 
 ```bash
 bundle install
-./scripts/geo apply resources/step_deployer.rb
+./scripts/geo apply resources/step-deployer.rb
```

Comment:
./scripts/geo doesn't actually exist in this repo

```
$ ./scripts/geo apply resources/step-deployer.rb
-bash: ./scripts/geo: No such file or directory
```

Reply:
that should be `bundle exec geo apply resources/step-deployer.rb`